extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmConfig.java

```diff
@@ -78,40 +78,40 @@
     /**
      * The size of the batches used when loading entities and collections.
      *
-     * `-1` means batch loading is disabled. This is the default.
+     * `0` means batch loading is disabled. This is the default.
      *
      * @asciidoclet
      */
-    @ConfigItem(defaultValue = "-1")
```

Comment:
This needs extra attention. I don't see any reason to use `-1` here but maybe I missed something.

Reply:
Maybe it should be an `OptionalInt`?